equinix/provider.go

```diff
@@ -205,19 +204,6 @@ func configureProvider(ctx context.Context, d *schema.ResourceData, p *schema.Pr
 	return &config, nil
 }
 
-// stringsFound returns true if all strings in source are found in target
-// Deprecated: stringsFound is shared between provider, tests, and resources
-// and is being relocated for package refactoring.
-// Use github.com/equinix/terraform-provider-equinix/internal/comparisons.Subsets
-func stringsFound(source []string, target []string) bool {
```

Comment:
This commit could be renamed from "refactor: move stringsFound and slicesMatch to comparisons package" to "refactor: remove stringsFound and slicesMatch (use comparisons pkg)"